packages/pla/pla.2.1/opam

```diff
@@ -0,0 +1,24 @@
+opam-version: "2.0"
+maintainer: "Leonardo Laguna Ruiz <modlfo@gmail.com>"
+synopsis: "Pla is a simple library and ppx syntax extension to create composable templates based on verbatim strings"
+homepage: "https://modlfo.github.io/pla/"
+bug-reports: "https://github.com/modlfo/pla/issues"
+license: "MIT"
+dev-repo: "git+https://github.com/modlfo/pla.git"
+
+build: [["dune" "build" "-p" name "-j" jobs]]
+
+depends: [
+  "ocaml" { >= "4.07.0" }
+  "dune" { build & >= "2.5" }
```

Comment:
Dune cannot be tagged as a build dependency (see https://github.com/ocaml/dune/issues/2147)
```suggestion
  "dune" { >= "2.5" }
```

Reply:
This change got removed in your latest push-force

Reply:
I have applied the suggested change.